cgroup2/manager_test.go

```diff
@@ -194,21 +199,35 @@ func TestKill(t *testing.T) {
 
 func TestMoveTo(t *testing.T) {
 	checkCgroupMode(t)
-	manager, err := NewManager(defaultCgroup2Path, "/test1", ToResources(&specs.LinuxResources{}))
+
+	manager, err := NewManager(defaultCgroup2Path, "/test-moveto-1", ToResources(&specs.LinuxResources{}))
 	if err != nil {
 		t.Error(err)
 		return
 	}
-	proc := os.Getpid()
+	defer func() {
+		_ = manager.Kill()
+		_ = manager.Delete()
+	}()
+
+	cmd := exec.Command("sleep", "infinity")
+	if err := cmd.Start(); err != nil {
+		t.Fatal(err)
+	}
+	proc := cmd.Process.Pid
 	if err := manager.AddProc(uint64(proc)); err != nil {
 		t.Error(err)
 		return
 	}
-	destination, err := NewManager(defaultCgroup2Path, "/test2", ToResources(&specs.LinuxResources{}))
+	destination, err := NewManager(defaultCgroup2Path, "/test-moveto-2", ToResources(&specs.LinuxResources{}))
```

Comment:
nit: can we rename the paths as destination

Reply:
You mean rename the destination variable NewManager returns or add destination somewhere in /test-moveto?

Reply:
```suggestion
	destination, err := NewManager(defaultCgroup2Path, "/dst-cgroup", ToResources(&specs.LinuxResources{}))
```

Reply:
Soemthing like these to make the test clearer?

Reply:
I like having the test prefix and something denoting what test, as if there is another leak like this it's pretty easy to deduce where it came from.

Reply:
Although test-moveto-dst and test-moveto-src are fine with me